easybuild/easyconfigs/m/MEGAN/MEGAN-6.22.0-Java-17.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Binary'
+
+name = 'MEGAN'
+version = '6.22.0'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'http://megan.informatik.uni-tuebingen.de/'
+description = "MEGAN is a comprehensive toolbox for interactively analyzing microbiome data"
+
+toolchain = SYSTEM
+
+local_installer = '%%(name)s_Community_unix_%s.sh' % version.replace('.', '_')
+source_urls = ['https://software-ab.informatik.uni-tuebingen.de/download/%(namelower)s%(version_major)s']
+sources = [local_installer]
+checksums = ['f9122124a470179dec2306bf5e59c6b6dd80843f8c11a268b3a036af709731fb']
+
+dependencies = [('Java', '17')]
+
+install_cmd = 'chmod +x %s && ./%s -q -dir %%(installdir)s' % (local_installer, local_installer)
+
+postinstallcmds = ["rm %(installdir)s/uninstall"]
+
+sanity_check_paths = {
+    'files': ['MEGAN'],
+    'dirs': ['jars', 'jars2', 'tools'],
+}
+
```

Comment:
Would it make sense to use one of the tools from the `tools` dir as a sanity check command? From what I saw in my test installation, these are bash scripts that call the same jar file as the `MEGAN` executable itself. And in contrast to `MEGAN`, these tools do seem to have a `--help`/`--version`, so perhaps it's still a good check to find out if the jar file can be called?